packages/clerk-js/src/ui/components/Checkout/CheckoutForm.tsx

```diff
@@ -6,17 +6,90 @@ import type {
   ClerkAPIError,
   ClerkRuntimeError,
 } from '@clerk/types';
-import { PaymentElement, useElements, useStripe } from '@stripe/react-stripe-js';
+import { Elements, PaymentElement, useElements, useStripe } from '@stripe/react-stripe-js';
+import type { Stripe } from '@stripe/stripe-js';
 import { useCallback, useEffect, useMemo, useState } from 'react';
 
-import { Button, Col, Flex, Form, Icon, Text } from '../../customizables';
-import { Alert, Disclosure, Divider, Select, SelectButton, SelectOptionList } from '../../elements';
+import { Box, Button, Col, descriptors, Flex, Form, Icon, Text } from '../../customizables';
+import { Alert, Disclosure, Divider, Drawer, LineItems, Select, SelectButton, SelectOptionList } from '../../elements';
 import { useFetch } from '../../hooks';
 import { ArrowUpDown, CreditCard } from '../../icons';
 import { animations } from '../../styledSystem';
 import { handleError } from '../../utils';
 
 export const CheckoutForm = ({
+  stripe,
+  checkout,
+  onCheckoutComplete,
+}: {
+  stripe: Stripe | null;
+  checkout: __experimental_CommerceCheckoutResource;
+  onCheckoutComplete: (checkout: __experimental_CommerceCheckoutResource) => void;
+}) => {
+  const { plan, planPeriod, totals } = checkout;
+  return (
+    <Drawer.Body>
+      <Box
+        elementDescriptor={descriptors.checkoutFormLineItemsRoot}
+        sx={t => ({
+          padding: t.space.$4,
+          borderBottomWidth: t.borderWidths.$normal,
+          borderBottomStyle: t.borderStyles.$solid,
+          borderBottomColor: t.colors.$neutralAlpha100,
+        })}
+      >
+        <LineItems.Root>
+          <LineItems.Group>
+            <LineItems.Title>{plan.name}</LineItems.Title>
+            <LineItems.Description suffix={`per month${planPeriod === 'annual' ? ', times 12 months' : ''}`}>
```

Comment:
{/* TODO(@COMMERCE): needs localization */}
```suggestion
            <LineItems.Description suffix={`per month${planPeriod === 'annual' ? ', times 12 months' : ''}`}>
```